cves/CVE-2011-1184.yml

```diff
@@ -159,14 +169,21 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: |
+     They have a good authenticator which works, but they should have taken the 
+     next step to see what happens if a user was able to find a authenticated
+     message. Another layer of protection just in case an attacker got past their 
+     authentication. 
```

Comment:
Is this even possible? If someone manages to get your authentication token, how could you differentiate them from the original person?
"find a authenticated" -> "find an authenticated"

Reply:
This is where nonce value is used to check if the message was old or already used.